utilities/pleiades/launch-dask-worker.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+#PBS -N dask-worker
+#PBS -W group_list=g26209
+#PBS -l select=1:ncpus=20:mpiprocs=5:model=ivy
+#PBS -l walltime=7:59:00
+#PBS -j oe
+#PBS -m abe
+
+export OMP_NUM_THREADS=4 
```

Comment:
I generally recommend setting this to 1, otherwise Dask.array can call parallelized operations in parallel again, leading to over-saturation.